x-pack/plugins/observability_solution/apm/server/routes/assets/services/get_service_assets.ts

```diff
@@ -52,6 +62,30 @@ export async function getServiceAssets({
           },
         };
       });
+
+      const { logsServiceNames, tracesServiceNames } = getServiceNamesPerSignalType(services);
+
+      const [apmMetrics, logsMetrics] = await Promise.all([
+        getServicesTransactionStats({
+          apmEventClient,
+          start,
+          end,
+          kuery,
+          serviceNames: tracesServiceNames,
+        }),
+        logsDataAccessStart.services.getLogsRatesService({
+          esClient,
+          identifyingMetadata: 'service.name',
+          timeFrom: start,
+          timeTo: end,
+          serviceNames: logsServiceNames,
+        }),
+      ]);
+
+      return services.map((item) => ({
+        ...item,
+        metrics: { ...apmMetrics[item.service.name], ...logsMetrics[item.service.name] },
```

Comment:
Now the `/services` returns both the names and the summar metrics but I'm a bit concerned about using the same approach with logs (or any other signal) because I'm not sure how performant querying log indices would be.


The `assets/services` is expected to be really fast. We could render the table with the names and progressively request the metrics.

Reply:
The problem of this is that the user won't be able to sort the table based on the logs summary values, right?